docs/dg/learningBasics.md

```diff
@@ -0,0 +1,416 @@
+
+**Commits Panel**
+
+1. Open a [report](https://dashboard-1507-pr-reposense-reposense.surge.sh/).
+2. Randomly open the commits panel of an author.
+3. Select a random commit in the commits panel, and hover your mouse on the icons on the commit title, there is no tooltip shown. For example, given the [report](https://dashboard-1507-pr-reposense-reposense.surge.sh/?search=&sort=groupTitle&sortWithin=title&since=&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=zoom&zA=eugenepeh&zR=reposense%2FRepoSense%5Bmaster%5D&zACS=99.28792569659443&zS=2017-10-09&zFS=&zU=2021-04-04&zMG=undefined&zFTF=commit&zFGS=groupByRepos&zFR=false) if you hover the mouse over the commit title `README: add acknowledgements section (#978)`, you should see a tooltip saying `Click to hide file details`, there is no corresponding tip shown.
+
+**Your Task** 
+
+Make corresponding `zoom panel` so that in step 3, when hovering your mouse over the commit title, a tooltip saying `Click to view the detailed file changes` will show up on the commit title.
+
+<panel header="Hint 1">
+
+Try to locate where the commit title is in [`zoom.pug`](https://github.com/reposense/RepoSense/blob/master/frontend/src/tabs/zoom.pug).
+</panel>
+
+<panel header="Hint 2">
+
+You can check how tooltip is added other icons in [`zoom.pug`](https://github.com/reposense/RepoSense/blob/master/frontend/src/tabs/zoom.pug).
+</panel>
+
+<panel header="Hint 3">
+
+Check what `selectedCommits` does and how the link and commit title of each commit is retrieved.
+</panel>
+
+<panel header="Suggested solution">
+  There is more than 1 way to achieve this. One solution is shown as the following:
+  
+  In the [indivdual commit section](https://github.com/reposense/RepoSense/blob/9125d4b6bea1e5bb6329cb8f7c476ea18fed3cea/frontend/src/tabs/zoom.pug#L88-L98) of `zoom.pug`, replace line 96 to 98 with the following [content](https://gist.github.com/HCY123902/e505429c26330df3ee17b1cb69972302/revisions) (See the latest revision).
+</panel>
+
+</panel>
+
+<box type="info" seamless>
+
+This is only for your practice. There is no need for you to commit this change and submit it in a pull request.
+</box>
+
+{{ step(4) }} Get familiar with the workflow
+
+The next step is to learn how to enhance the code quality according to the coding standard, write necessary tests and documentation, verify that the changes passes the test cases, and submit the pull request. You can refer to [the _Workflow_ page](workflow.html).
+
+{{ step(5) }} Get started with contributing
+
+RepoSense currently has a lot of open issues, and some may be challenging. Now that you are ready, you can check the [issues for first timers](https://github.com/reposense/RepoSense/issues?q=is%3Aopen+is%3Aissue+label%3Ad.FirstTimers+label%3Aa-FrontEnd) or issues in the [roadmap for contributors](https://github.com/reposense/RepoSense/projects/2#column-13522966) to work on actual pull requests.
```

Comment:
These two steps should not be here as they are not part of 'learning the basics'. Those two steps are in a different page. You can add more details to that page if you want.

Reply:
Thanks for the suggestion. I will move it to the workflow page accordingly.